README.md

```diff
@@ -38,6 +41,14 @@ Once the migration is complete, check configuration by starting the server. Star
 
 ![screenshot_home.jpg](./images/screenshot_home.png)
 
+## Running Docker locally
+
+```
+cp deployment/Dockerfile .
+docker build . -t umf
+docker run --env-file .env -p 2998:2998 --entrypoint bash -it umf
+```
+
```

Comment:
I think we need to add details to this section. The following are not obvious to me:

- Is this an _alternative_ to following the instructions above (in Getting Started)?
- If so, what's next, after opening a bash session in the container? What do I do in the terminal?
- What do I put in my `.env` file?
- Is the intent to have docker connect to postgres running outside of the container, on the host? If so, this definitely needs details (or at least a link to relavent instructions online)